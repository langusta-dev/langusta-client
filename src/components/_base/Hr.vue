<template>
  <div _h=".3" _bg="primary-contrast/30" _rounded />
</template>
