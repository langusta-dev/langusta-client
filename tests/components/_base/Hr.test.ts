import Hr from '~/components/_base/Hr.vue';

import { mount } from '@vue/test-utils';

describe('Hr', () => {
  it('should render', () => {
    const wrapper = mount(Hr);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
