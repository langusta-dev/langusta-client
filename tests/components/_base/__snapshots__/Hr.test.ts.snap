// Vitest Snapshot v1

exports[`Hr > should render 1`] = `"<div _h=\\".3\\" _bg=\\"primary-contrast/30\\" _rounded=\\"\\"></div>"`;
